curriculum/challenges/english/01-responsive-web-design/accessibility-quiz/part-067.md

```diff
@@ -0,0 +1,689 @@
+---
+id: 6148e5aeb102e3142de026a2
+title: Part 67
+challengeType: 0
+dashedName: part-67
+---
+
+# --description--
+
+Finally, the navigation accessibility can be improved with providing keyboard shortcuts.
+
+The `accesskey` attribute accepts a space-separated list of access keys. For example:
+
+```html
+<button type="submit" access-key="s">Submit</button>
+```
+
+Give each of the navigation links a single-letter access key.
+
+_Note: It is not always advised to use access keys, but they can be useful_
+
+Well done. You have completed the _Accessibility Quiz_ practice project.
+
+# --hints--
+
+You should give the first `a` element a single-letter `accesskey`.
+
+```js
+assert.equal(document.querySelectorAll('a')?.[0]?.getAttribute('accessKey')?.length, 1);
+```
+
+You should give the second `a` element a single-letter `accesskey`.
+
+```js
+assert.equal(document.querySelectorAll('a')?.[1]?.getAttribute('accessKey')?.length, 1);
+```
+
+You should give the third `a` element a single-letter `accesskey`.
+
+```js
+assert.equal(document.querySelectorAll('a')?.[2]?.getAttribute('accessKey')?.length, 1);
+```
```

Comment:
````suggestion
assert.equal(document.querySelectorAll('a')?.[0]?.getAttribute('access-key')?.length, 1);
```

You should give the second `a` element a single-letter `accesskey`.

```js
assert.equal(document.querySelectorAll('a')?.[1]?.getAttribute('access-key')?.length, 1);
```

You should give the third `a` element a single-letter `accesskey`.

```js
assert.equal(document.querySelectorAll('a')?.[2]?.getAttribute('access-key')?.length, 1);
```
````

Oops did I suggest this wrong last time?

Reply:
I have tested this so many times, I am now just confused.

Reply:
`access-key` also does not exist. `accesskey` does, but so should `accessKey`, because: https://developer.mozilla.org/en-US/docs/Web/API/Element/getAttribute#lower_casing

Reply:
This might have been my fault because the attribute is `accesskey` not `access-key`